demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -232,12 +232,21 @@ interface Toggle {
 }
 
 interface TranscriptSegment {
-  content: string;
+  contentSpan: HTMLSpanElement,
   attendee: Attendee;
   startTimeMs: number;
   endTimeMs: number;
 }
 
+interface TranscriptionStreamParams {
+  contentIdentificationType?: 'PII' | 'PHI';
+  contentRedactionType?: 'PII';
+  enablePartialResultsStability?: boolean;
+  partialStabilityFactor?: string;
```

Comment:
Please rename to `partialResultsStability` as per the official [Transcribe API](https://docs.aws.amazon.com/transcribe/latest/dg/API_streaming_StartStreamTranscription.html#API_streaming_StartStreamTranscription_RequestSyntax).